tests/integration/utilities/test_prometheus_collectors.py

```diff
@@ -99,17 +122,14 @@ def test_staking_provider_metrics_collector(test_registry, staking_providers):
         eth_endpoint=MOCK_ETH_PROVIDER_URI,
     )
     collector_registry = CollectorRegistry()
-    prefix = "test_staking_provider_metrics_collector"
-    collector.initialize(metrics_prefix=prefix, registry=collector_registry)
+    collector.initialize(registry=collector_registry)
     collector.collect()
 
     taco_application_agent = ContractAgency.get_agent(
         TACoApplicationAgent, registry=test_registry
     )
 
-    active_stake = collector_registry.get_sample_value(
-        f"{prefix}_associated_active_stake"
-    )
+    active_stake = collector_registry.get_sample_value("associated_active_stake")
```

Comment:
See related suggestion about renaming this metric.

```suggestion
    active_stake = collector_registry.get_sample_value("active_stake")
```